ui/apps/platform/src/Containers/AccessControl/AccessControl.tsx

```diff
@@ -18,11 +19,60 @@ function AccessControl(): ReactElement {
     // TODO is read access required for all routes in improved Access Control?
     // TODO Is write access required anywhere in classic Access Control?
     const { hasReadAccess } = usePermissions();
-    const hasReadAccessForAuthProvider = hasReadAccess('Access');
+    const hasReadAccessForAccessControlPages = hasReadAccess('Access');
 
     return (
         <>
-            {hasReadAccessForAuthProvider ? (
+            <Alert
+                isInline
+                variant="warning"
+                title={
+                    <>
+                        <p>The following permission resources have been replaced:</p>
+                        <List>
+                            <ListItem>
+                                <b>Access</b> replaces{' '}
+                                <b>AuthProvider, Group, Licenses, and User</b>
+                            </ListItem>
+                            <ListItem>
+                                <b>DeploymentExtension</b> replaces{' '}
+                                <b>Indicator, NetworkBaseline, ProcessWhitelist, and Risk</b>
+                            </ListItem>
+                            <ListItem>
+                                <b>Integration</b> replaces{' '}
+                                <b>
+                                    APIToken, BackupPlugins, ImageIntegration, Notifier, and
+                                    SignatureIntegration
+                                </b>
+                            </ListItem>
+                            <ListItem>
+                                <b>Image</b> now also covers <b>ImageComponent</b>
+                            </ListItem>
+                        </List>
+
+                        <p>
+                            The following permission resources will be replaced in the upcoming
+                            versions:
+                        </p>
+                        <List>
+                            <ListItem>
+                                <b>Administration</b> will replace{' '}
+                                <b>
+                                    AllComments, Config, DebugLogs, NetworkGraphConfig, ProbeUpload,
+                                    ScannerDefinitions, SensorUpgradeConfig, and ServiceIdentity
```

Comment:
In the Changelog entry from PR 3036, `ScannerBundle` is also in the list of permissions replaced by `Administration`.